app/serializers/mentor_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class MentorSerializer
   include FastJsonapi::ObjectSerializer
-  attributes :id, :name, :email, :city, :state, :slack_username, :linkedin_username, :github_username, :program, :active, :matched
+  attributes :id, :name, :email, :city, :state, :slack_username, :active, :matched
 
```

Comment:
Do we want to specify which program a mentor is in?

Reply:
as a team we decided to defer this for now, since mentors do specify whether they prefer a BE or FE mentee, and in the description they can describe their past program. Can add later if clients need it.